File: app/src/main/java/com/norvera/confession/ui/prayers/PrayerDetail.java
package com.norvera.confession.ui.prayers;


import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;


import com.norvera.confession.data.models.PrayersEntry;
import com.norvera.confession.databinding.FragmentPrayerDetailBinding;

import androidx.fragment.app.Fragment;

/**
 * A simple {@link Fragment} subclass.
 */
public class PrayerDetail extends Fragment {


    public PrayerDetail() {
        // Required empty public constructor
    }


    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        PrayersEntry prayersEntry = PrayerDetailArgs.fromBundle(getArguments()).getPrayerEntry();
        FragmentPrayerDetailBinding binding = FragmentPrayerDetailBinding.inflate(inflater, container, false);

        binding.setPrayerEntry(prayersEntry);
        //binding.tvGuideDetail.setText(guideEntry.guideText);

        // Inflate the layout for this fragment
        return binding.getRoot();
    }

}

File: app/src/main/java/com/norvera/confession/ui/prayers/PrayerDetail.kt
package com.norvera.confession.ui.prayers


import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import com.norvera.confession.databinding.FragmentPrayerDetailBinding


/**
 * A simple [Fragment] subclass.
 */
class PrayerDetail : Fragment() {


    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        val prayersEntry = PrayerDetailArgs.fromBundle(arguments!!).prayerEntry
        val binding = FragmentPrayerDetailBinding.inflate(inflater, container, false)

        binding.prayerEntry = prayersEntry
        //binding.tvGuideDetail.setText(guideEntry.guideText);

        // Inflate the layout for this fragment
        return binding.root
    }

}// Required empty public constructor
